src/components/RequestPanel/TransactionProgress.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import { GU, ProgressBar, textStyle, useTheme } from '@aragon/ui'
+import useNow from '../../hooks/useNow'
+import { norm } from '../../lib/math-utils'
+import { getRelativeTime } from '../../utils/date-utils'
+
+const MINUTE = 60 * 1000
+const TX_DURATION_AVERAGE = 2 * MINUTE
+const TX_DURATION_THRESHOLD = TX_DURATION_AVERAGE - MINUTE / 2
+
+function TransactionProgress({ createdAt }) {
+  const theme = useTheme()
+  const now = useNow().valueOf()
```

Comment:
I believe we use `getTime()` in the client—let's keep doing that. Semantically, it's a bit more correct in intent (https://stackoverflow.com/a/9710267).

Reply:
Ah actually this is a `dayjs` instance  and it [uses `getTime` under the hood](https://github.com/iamkun/dayjs/blob/bcea06730da2d19ddbad844de3557ee077af821c/src/index.js#L158)